GrandCentralBoard/Widgets/Watch/WatchWidgetViewModel.swift

```diff
@@ -8,59 +8,119 @@ import UIKit
+    private static let eventNameFont = UIFont.systemFontOfSize(22, weight: UIFontWeightBold)
+    private static let calendarNameTextColor = UIColor(red: 35/255, green: 208/255, blue: 165/255, alpha: 1)
+    private static let highLineSpacingStyle: NSParagraphStyle = {
+        let style = NSMutableParagraphStyle()
+        style.lineSpacing = 34
+        style.alignment = .Center
+        return style
+    }()
+    private static let lowLineSpacingStyle: NSParagraphStyle = {
+        let style = NSMutableParagraphStyle()
+        style.lineSpacing = 12
+        style.alignment = .Center
+        return style
+    }()
 
-        let blinking = (currentTimeComponents.minute / 5) + 1
+    private static func descriptionForEvent(event: Event,
+                                            fromCalendar calendarName: String?,
+                                            startingInMinutes minutesToEvent: Int) -> NSAttributedString {
+        let resultString = NSMutableAttributedString()
 
-        blinkingImage = UIImage(named: "\(blinking)")
-        watchFaceImage = UIImage(named: "f\(blinking)")
+        if minutesToEvent > 0 {
+            let nextMeetingIn = NSAttributedString(
+                string: "NEXT MEETING IN" + "\n",
+                attributes: [NSFontAttributeName : nextMeetingInFont, NSParagraphStyleAttributeName : lowLineSpacingStyle]
+            )
 
-        if currentTimeComponents.minute < 30 {
-            hourLeft = nil
-            hourRight = "\(currentTimeComponents.hour % 24)"
+            let minutes = NSMutableAttributedString(
+                string: "\(minutesToEvent) MIN" + "\n",
+                attributes: [NSFontAttributeName : minutesFont, NSParagraphStyleAttributeName : lowLineSpacingStyle]
+            )
+            resultString.appendAttributedString(nextMeetingIn)
+            resultString.appendAttributedString(minutes)
         } else {
-            hourLeft = "\(currentTimeComponents.hour + 1 % 24)"
-            hourRight = nil
+            let now = NSAttributedString(
+                string: "NOW" + "\n",
+                attributes: [NSFontAttributeName : nowFont, NSParagraphStyleAttributeName : highLineSpacingStyle]
+            )
+            resultString.appendAttributedString(now)
         }
 
-        self.calendarName = calendarName?.uppercaseString
+        if let calendarName = calendarName {
+            let calendarNameString = NSMutableAttributedString(
+                string: "@\(calendarName.uppercaseString)" + "\n",
+                attributes: [NSFontAttributeName : calendarNameFont, NSForegroundColorAttributeName : calendarNameTextColor]
+            )
+            resultString.appendAttributedString(calendarNameString)
+        }
+
+        let eventNameString = NSMutableAttributedString(
+            string: event.name.uppercaseString,
+            attributes: [NSFontAttributeName : eventNameFont]
+        )
+        resultString.appendAttributedString(eventNameString)
```

Comment:
Note, that I'm setting the line height attribute for "newlined" strings:
```swift
let minutes = NSMutableAttributedString(
   string: "\(minutesToEvent) MIN" + "\n",
   attributes: [NSFontAttributeName : minutesFont, NSParagraphStyleAttributeName : lowLineSpacingStyle]
)
```
This gives me a simple control of the bottom padding to next line.

Because I need to apply different `lineSpacingStyle` attribute to each `"\n"` attributed string I can't simply use one representation of `"\n"` to join all items, like you suggest.